add information for logistical planning (#3)
digital-earths-global-hackathon/planning: hosting/logistics/index.qmd

---
title: Logistics for hosting a hackathon site
---

Thoughts about hackathon organization based on 4 1/2 nextGEMS hackathons.

## Participant numbers

* nextGEMS numbers:
  + Berlin, Oct 2021: 78 (17 online)
  + Vienna, Jul 2022: 106 
  + Madrid, May 2023: 134
  + Hamburg, Mar 2024: 137
  + Wageningen, Oct 2024: 71 registrations (*as of Jul 2024*)
* for the global hackathon, we expect 50--150 participants per location

## Agenda
this is the general schedule, from the Madrid hackathon onwards, which should allow most participants to travel on Monday and Friday

* Monday afternoon to Friday morning
* Monday afternoon: welcome, theme updates, general information
* Friday afternoon: work group roundup, general discussion of future steps for the project
* Tuesday to Thursday: group work
* some general discussions or keynote talks sprinkled through the week
* This is something you should decide on, how you would like to structure the week

> Here is a very general [**draft agenda**](agenda_template.qmd). Might be helpful to forward to potential venues.

Reviewer: ```suggestion
Here is a very general [**draft agenda**](agenda_template.qmd). Might be helpful for potential venues.
```
